index.html

```diff
@@ -217,21 +220,23 @@ <h1>Tonic</h1>
                     </div><br><br>
                     <div class="item desktop">
                         <img class="img" id="place1" src="images/Snapshoot Portfolio3.svg" alt="img">
-                        <div class="title-post" id="place2">
-                            <h1>Multi-Post Stories</h1>
-                        </div>
-                        <div class="tags" id="place3">
-                            <ul>
-                                <li class="canopy">Uber</li>
-                                <li class="canopy grey"><i class="fa-solid fa-circle"></i></li>
-                                <li class="canopy grey">Full Stack Developer</li>
-                                <li class="canopy grey"><i class="fa-solid fa-circle"></i></li>
-                                <li class="canopy grey">2015</li>
-                            </ul>
-                        </div>
                         <div class="right-block" id="place4">
+                            <div class="title-post" id="place2">
+                                <h1>Multi-Post Stories</h1>
+                            </div>
+                            <div class="tags" id="place3">
+                                <ul>
+                                    <li class="canopy">Uber</li>
+                                    <li class="canopy grey"><i class="fa-solid fa-circle"></i></li>
+                                    <li class="canopy grey">Full Stack Developer</li>
+                                    <li class="canopy grey"><i class="fa-solid fa-circle"></i></li>
+                                    <li class="canopy grey">2015</li>
+                                </ul>
+                            </div>
```

Comment:
- [ ] Kindly try to follow the design as much as possible, for the project cards, the margins, padding, and sizes are different from the ones on Figma.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194648177-53e7f8d6-d631-404e-bc99-37a2b26fa76e.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194648252-aa481fc4-7d84-484e-9162-f63453a87e78.png)